riscv/src/continuations/bootloader.rs

```diff
@@ -58,7 +58,9 @@ pub fn shutdown_routine_upper_bound(num_pages: usize) -> usize {
 
 pub fn bootloader_specific_instruction_names(field: KnownField) -> [&'static str; 2] {
     match field {
-        KnownField::BabyBearField | KnownField::Mersenne31Field => todo!(),
+        KnownField::BabyBearField | KnownField::KoalaBearField | KnownField::Mersenne31Field => {
+            todo!()
+        }
         KnownField::GoldilocksField | KnownField::Bn254Field => {
```

Comment:
And here?

Reply:
Right!

(I wasn't sure what we'll want to distinguish here in the future, but seems plausible that all small fields will share the same bootloader / shutdown routine.)

Also fixed the others.